xblockutils/public/studio_edit.js

```diff
@@ -51,17 +51,18 @@ function StudioEditableXBlockMixin(runtime, element) {
         if (type == 'html' && tinyMceAvailable) {
             tinyMCE.baseURL = baseUrl + "/js/vendor/tinymce/js/tinymce";
             $field.tinymce({
-                theme: 'modern',
-                skin: 'studio-tmce4',
+                theme: 'silver',
+                skin: 'studio-tmce5',
+                content_css: 'studio-tmce5',
                 height: '200px',
                 formats: { code: { inline: 'code' } },
                 codemirror: { path: "" + baseUrl + "/js/vendor" },
                 convert_urls: false,
-                plugins: "link codemirror",
+                plugins: "lists, link, codemirror",
                 menubar: false,
                 statusbar: false,
                 toolbar_items_size: 'small',
-                toolbar: "formatselect | styleselect | bold italic underline forecolor wrapAsCode | bullist numlist outdent indent blockquote | link unlink | code",
```

Comment:
`wrapAsCode` is [a custom button](https://github.com/openedx/edx-platform/blob/54507c1e083990ef9d9c8022027332818bd05595/xmodule/js/src/html/edit.js#L1246-L1256). Since it has not been working before, we can remove it.